packages/docs/pages/components/Card.mdx

```diff
@@ -0,0 +1,58 @@
+import Card from '../../components/Card/Card';
+
+# Card
+
+Is a container to have a header and a content ando show detail information about a specific element. You can use the [paper props](/platzily-ui/components/Paper) in the cards
```

Comment:
`It's a container that have a header and a content, it shows detailed information about an specific element.` The other part is great :grin: